openapi/kas-fleet-manager.yaml

```diff
@@ -921,15 +932,22 @@ components:
             $ref: "#/components/schemas/ObjectReference"
     Error:
       allOf:
-        - $ref: "#/components/schemas/ObjectReference"
         - type: object
+          required: [id, kind, href, code, reason]
```

Comment:
I think we can revert this part as rule is disabled

Reply:
let's see if we can find ways to come with a better schema that works with the error check enabled.
I'll prefer if we can deal with quick wins right away rather than disabling the checks.